system/Model.php

```diff
@@ -1374,13 +1346,13 @@ public function setValidationMessage(string $field, array $fieldMessages)
 	 * Should validation rules be removed before saving?
 	 * Most handy when doing updates.
 	 *
-	 * @param boolean $choice
+	 * @param boolean $clean
 	 *
 	 * @return $this
 	 */
-	public function cleanRules(bool $choice = false)
+	public function cleanRules(bool $clean = true)
```

Comment:
change default value of true/false is bc break, please revert.

Reply:
please check
insert and insertBatch methods $this->cleanRules(false), 
update and updateBatch methods $this->cleanRules()
replace method $this->cleanRules()

I think it doesn't bc break

methods its

Reply:
it is public method, and public method can be used in extended class. If the class that extends Model call cleanRules without parameter, it will make different result.

Reply:
I thought it will return the default propriety protected $cleanValidationRules = true;

Reply:
no, with default value in method parameter like this, it will replace it with assignment. see: https://3v4l.org/0hmeY for the proof

Reply:
please check this https://3v4l.org/qL7Ml and then i will revert

Reply:
The problem is the existing user that already call the method **without** passing any parameter in his/her application in the extended class(es), and he/she run composer update, he/she will got different result as it have different default value.

Reply:
reverted and thank you for your patience